test/utils/seeder.ts

```diff
@@ -11,6 +11,8 @@ export const createUser = ({
   name = faker.name.findName(),
   avatar = faker.internet.avatar(),
   roles = [],
+  channels = [],
+  available = false,
```

Comment:
Just curious, is there any reason why we're defaulting to `false`?

Reply:
As opposed to `true`?  Not really, no.  `true` should be the more common case, so I'll default to `true` instead; great point!